_posts/2019-08-18-view-encapsulation-w-angularze.md

```diff
@@ -0,0 +1,426 @@
+---
+layout:    post
+title:     "View Encapsulation w Angularze - czyli o kapsułkowaniu słów kilka"
+published: true
+date:     2019-08-18 08:00:00 +0100
+author:    mhoja
+tags:
+    - angular
+    - view encapsulation
+    - kapsułkowanie
+---
+
+Tworząc komponenty w Angularze mamy możliwość zarządzania kapsułkowaniem styli - czyli tym jak style z jednego komponentu wpływają na inne komponenty.  
+Zanim omówimy kapsułkowanie, wtyjaśnijmy w kilku słowach czym jest **Shadow DOM**.
+
+## Shadow DOM
+
+**Shadow DOM** wprowadza enksapsulację do DOM'u. Pozwala to odseparować styl i kod potrzebny do wyświetlenia od dokumentu, w którym znajduje się dany element. Przykładem może być np. znacznik HTML `<video>`
```

Comment:
DOM-u
odmieniając skrótowce używamy myślników

Reply:
raz używasz słowa enkapsulacja, a raz kapsułkowania, uspójnijmy to

Reply:
> Znacznik HTML `<video>`

raczej element HTML

Reply:
done